post.js

```diff
@@ -126,7 +173,9 @@ class DistributedPost extends HTMLElement {
 
   renderPostContent(jsonLdData) {
     // Clear existing content
-    this.innerHTML = "";
+    while (this.firstChild) {
```

Comment:
I think in this case, `.innerHTML` is actually cleaner. It's just when we add user supplied text that it's dangerous.

Reply:
Yeah, sure!